### PR TITLE
PEAR-1935 - Alternatives to complex images - Box plot/QQ plot

### PR DESCRIPTION
**Pull request by amlehman**
## Description
\+ PEAR-1936 - Alternatives to complex images - Venn diagram
Converts the Venn Diagram component to use Apache ECharts instead of Plotly because Plotly offers 0 accessibility options :(

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

File: packages/portal-proto/src/features/set-operations/SetOperations.tsx [hunk 1]
```diff
@@ -46,7 +46,8 @@ export const SetOperations: React.FC<SetOperationsProps> = ({
       </div>
       <div className="flex flex-row pt-2">
         <VennDiagram
-          labels={["S<sub>1</sub>", "S<sub>2</sub>", "S<sub>3</sub>"]}
+          labels={["S₁", "S₂", "S₃"]}
+          ariaLabel="The Venn diagram displays the number of cases shared between the cohorts."
```
Comment (wteouchicago) [inline packages/portal-proto/src/features/set-operations/SetOperations.tsx]: this label is for Cohort Comparison. set operations' should be:
`The Venn diagram displays the intersections, unions, or differences in the cohorts or sets. Additional information can be found in the Summary Table and the Overlap Table.`

File: packages/portal-proto/src/features/cohortComparison/CohortVennDiagram.tsx [hunk 2]
```diff
@@ -59,7 +59,8 @@ const CohortVennDiagram: React.FC<CohortVennDiagramProps> = ({
           highlighted: false,
         },
       ]}
-      labels={["S<sub>1</sub>", "S<sub>2</sub>"]}
+      labels={["S₁", "S₂"]}
+      ariaLabel="The Venn diagram displays the intersections, unions, or differences in the cohorts or sets. Additional information can be found in the Summary Table and the Overlap Table."
```
Comment (wteouchicago) [inline packages/portal-proto/src/features/cohortComparison/CohortVennDiagram.tsx]: this label's for set operations. for cohort comparison, it should be 
`The Venn diagram displays the number of cases shared between the cohorts.`

File: packages/portal-proto/src/features/cohortComparison/CohortCard.tsx [hunk 3]
```diff
@@ -72,6 +72,20 @@ const CohortCard: React.FC<CohortCardProps> = ({
...
+        <CohortVennDiagram caseSetIds={caseSetIds} cohorts={cohorts} />
+      ) : (
+        <VennDiagram
+          chartData={[
+            { key: "S1_minus_S2", value: 0, highlighted: false },
+            { key: "S2_minus_S1", value: 0, highlighted: false },
+            { key: "S1_intersect_S2", value: 0, highlighted: false },
+          ]}
+          labels={["S₁", "S₂"]}
+          ariaLabel="The Venn diagram displays the intersections, unions, or differences in the cohorts or sets. Additional information can be found in the Summary Table and the Overlap Table."
```
Comment (wteouchicago) [inline packages/portal-proto/src/features/cohortComparison/CohortCard.tsx]: label for cohort comparison should be:
`The Venn diagram displays the number of cases shared between the cohorts.`

**Comment by wteouchicago** [comment]
Only taking a quick look. Evgeniia can UAT more when it's merged.

cDAVE changes look good.

## Issues related to Venn Diagram in Set Ops and Cohort Comparison

1. Minor. Can we comma-separate thousands for the numbers in the Venn diagram In Set Ops?

2. I'm comparing Set Operations demo from this PR vs develop. In the Venn diagram for this PR, the (S1 and S3 intersection) and the (S3 only) sections have different counts from develop (swapped). They are also different from the counts in the table. See screenshot. Is this correct? 

Screenshot 2024-06-03 at 10 23 09 PM

3. Can the number subscripts in the labels for the Venn diagrams be a little larger? :squint:

4. aria labels for the Venn diagrams in Cohort Comparison vs Set Operations seem switched around.

5. Cohort Comparison demo mode and regular mode -- no numbers are showing up in the Venn diagram for me.
Screenshot 2024-06-03 at 10 39 34 PM

- I do see what looks like the correct response from the API
- this seems to be happening in develop as well. I can file a separate ticket, lemme know.


6. Prob. need @cemile-ctds's eyes on this, but the "View Venn diagram in Set Operations" link looks a little detached from the Venn diagram and seems more related to the checkboxes below. If we can't bring it nearer to the Venn diagram, maybe we need to increase the space between the link and the checkboxes below? 🤷‍♀️

**Comment by wteouchicago** [comment]
Issue 5 is OK in Pari's PR (bugfix/PEAR-1787-Version-3-loggedIn-injection), so maybe we just need to wait for it to be merged? cc @paribartandhakal

**Comment by amlehman** [comment]
Hi @wteouchicago, yeah the issue is actually a bug from PEAR-1860 and Pari noticed it and fixed it. I applied the fix on this branch since it was small. All of your feedback points should be addressed, let me know if you want font/spacing further adjusted.

**Comment by wteouchicago** [comment]
looks good! 👍

**Review by craigrbarnes — APPROVED** [review]
Changes work and code looks fine

**Pull request merged**